Standards/CCSMB-9.md

```diff
@@ -0,0 +1,38 @@
+# CCSMB 9: Disabling Automatic Updating and Dependency Installation in Lua Programs
+*Author: Tomodachi94 <@tomodachi94>*
+
+## Rationale
+This RFC allows users to programmatically disable autoupdating and dependency installation in programs.
+Some users opt to use package managers when installing programs, which typically manage updates and dependencies on their own; additionally, some users would like to remain on a specific version for stability.
+
+## Autoupdating specification
+Programs implementing an autoupdater MAY support the following options for disabling that function.
+
+Programs which are specifically designed to update programs, namely installers and package managers, SHOULD NOT implement the following.
+
+Programs implementing an autoupdater MUST implement all options listed below in order to be considered compliant.
+
+### `settings` API setting
+Compliant programs MUST allow disabling autoupdating through the `autoupdating.disableAutoupdating` setting. If the option is set to `true`, autoupdating must be disabled.
```

Comment:
Wouldn't naming the setting `autoupdate.enable` (in the style of the already existing `motd.enable`) be more convenient? Of course, it would be considered as set to `true` by default.

Reply:
yes, it should be since usually CC stuff is group.enable rather than group.disableFeature

Reply:
Resolved in e784fccf31e5fbb2ef39ef50a9e74926aad06930